Operations/pycpt-operational.ipynb

```diff
@@ -644,7 +645,8 @@
    "outputs": [],
    "source": [
     "point_latitude = None\n",
-    "point_longitude = None"
+    "point_longitude = None\n",
+    "color_bar='DL_CORRELATION_COLORMAP'"
```

Comment:
I suggest that the notebook as found in github should say `color_bar = None`, so that the default will be used, and the documentation (see previous cell) should explain that if you want a different colorbar you can set `color_bar` to one of the other options (and explain where to find the list of options).

Reply:
Done